packages/n3h-mock/lib/index.js

```diff
@@ -478,19 +536,38 @@ class N3hMock extends AsyncClass {
   /**
    *
    */
-  _track (dnaAddress, agentId, fromZmqId) {
+  _untrack (dnaAddress, agentId) {
+    // get mem slice
     const ref = this._getMemRef(dnaAddress)
+    // create data entry
+    const agent = {
+      type: 'agent',
+      dnaAddress: dnaAddress,
+      agentId: agentId,
+      transportId: undefined
+    }
+    log.t('_untrack() for "' + agentId + '" for DNA "' + dnaAddress + '"')
+
     // store agent (this will map agentId to transportId)
-    ref.mem.insert({
+    ref.mem.insert(agent)
```

Comment:
Unlike hackmode where we've got nodes gossiping and re-introducing the data if it's gone... for mock mode, we could actually delete it. I'm totally fine with this solution, though if you'd prefer to keep it this way.

Reply:
yeah totally. I did it like this because it was easier / faster to do, and because I knew in hackmode I wouldn't be able to do a delete. n3h-mock is kind of my sandbox / training wheels before doing hackmode :)